### PR TITLE
[DOCS 7045] Alfresco Content Services 7.3 Install with Ansible updates

### PR DESCRIPTION
**Pull request by Amy-Hyland**

### CODE REVIEW

File: content-services/latest/install/ansible.md [hunk 1]
```diff
@@ -61,110 +63,104 @@ all:
...
+|-|-|-|-|-|-|-|
+| Amazon Linux (v2) | - | - | - | - | - | - |
+| Amazon Linux (v1) | - | - | - | - | - | - |
+| RHEL 8.6 | [x] | - | - | - | - | [x] |
+| RHEL 8.5 | [x] | [x] | - | - | - | [x] |
+| RHEL 8.4 | [x] | [x] | [x] |[x] | - | [x] |
+| RHEL 8.2 | [x] | [x] | [x] | - | [x] |
+| RHEL 7.7 | - | - | [x] | [x] | - | [x] |
+| RHEL 7.6 | - | - | [x] | [x] | [x] | [x] |
+| CentOS 8 x64 | [x] | [x] |[x] |[x] | - | [x] |
```
Comment (alxgomz) [inline content-services/latest/install/ansible.md:108]: That's a mistake on our side. Centos 8 is not supported anymore for 7.3. You can actually remove the line completely as we have removed support for it in the latest playbook version. If a customer wants to deploy on centos 8, he would need to use an older version of the playbook

File: content-services/latest/install/ansible.md [hunk 2]
```diff
@@ -61,110 +63,104 @@ all:
...
 
 ## Set up Ansible
-Not all distributions of Linux may match the version requirements for Ansible and its dependencies. Below we describe 
-how to configure a control node  with one of the many ways to set a python virtual environment. With python `virtualenvs` 
-you can install the exact same versions of Ansible we use when testing without impacting your system installation of python. 
-Doing so you're ensuring best chances of success.
+
+Not all distributions of Linux may match the version requirements for Ansible and its dependencies. Below we describe how to configure a control node  with one of the many ways to set a python virtual environment. With python `virtualenvs` you can install the exact same versions of Ansible we use when testing without impacting your system installation of python. Doing so you're ensuring best chances of success.
 
 1. Download the Ansible playbook [zip file](https://nexus.alfresco.com/nexus/service/local/repositories/releases/content/org/alfresco/alfresco-ansible-deployment/2.0.0/alfresco-ansible-deployment-2.0.0.zip)
```
Comment (alxgomz) [inline content-services/latest/install/ansible.md:144]: That version old... please match up to the latest release tagged in the repo: 2.2.1.
For 7.4. We'll be releasing 2.3.0

File: content-services/latest/install/ansible.md [hunk 3]
```diff
@@ -61,110 +63,104 @@ all:
...
 
-2. If you're not working directly on the control node, transfer the ZIP file to the control node together with the 
-   SSH private key required to login to the target machines, and SSH into the machine
+2. If you're not working directly on the control node, transfer the ZIP file to the control node together with the SSH private key required to login to the target machines, and SSH into the machine
```
Comment (alxgomz) [inline content-services/latest/install/ansible.md:146]:
```suggestion
If you're not working directly on the control node, transfer the ZIP file to the control node together with the SSH private key required to login to the target machines, and SSH into the control node
```

File: content-services/latest/install/ansible.md [hunk 4]
```diff
@@ -61,110 +63,104 @@ all:
...
     scp  alfresco-ansible-deployment-<version>.zip user@controlnode:
     scp  ~/.ssh/ansible_rsa user@controlnode:.ssh
     ssh  user@controlnode
     ```
 
-    >**Note:** You may want to generate an SSH key pair locally and use it later for deployment. Whether you use a locally
-    >generated key, or copy over a key to the control node, it is your responsibility to deploy it to the target machines 
-    >so Ansible can use it. Using SSH keys is recommended but not mandatory. If you instead use password, make sure to add 
-    >the `-k`switch to the ansible command, so it prompts you for a password.
+    >**Note:** You may want to generate an SSH key pair locally and use it later for deployment. Whether you use a locally generated key, or copy over a key to the control node, it is your responsibility to deploy it to the target machines so Ansible can use it. Using SSH keys is recommended but not mandatory. If you instead use password, make sure to add the `-k`switch to the ansible command, so it prompts you for a password.
```
Comment (alxgomz) [inline content-services/latest/install/ansible.md:154]:
```suggestion
>**Note:** You may want to generate an SSH key pair locally and use it later for deployment. Whether you use a locally generated key, or copy over a key to the control node, it is your responsibility to deploy it to the target machines so Ansible can use it. Using SSH keys is recommended but not mandatory. If you instead use password, make sure to add the `-k` switch to the `ansible` command, so it prompts you for a password
```

File: content-services/latest/install/ansible.md [hunk 5]
```diff
@@ -206,6 +256,7 @@ ansible-playbook playbooks/acs.yml -i inventory_local.yml
 
 Alternatively, to deploy other versions of Content Services use one of the following commands:
 
+* **7.2**: `ansible-playbook playbooks/acs.yml -i inventory_local.yml -e "@7.2.N-extra-vars.yml"`
 * **7.1**: `ansible-playbook playbooks/acs.yml -i inventory_local.yml -e "@7.1.N-extra-vars.yml"`
 * **7.0**: `ansible-playbook playbooks/acs.yml -i inventory_local.yml -e "@7.0.N-extra-vars.yml"`
 * **6.2**: `ansible-playbook playbooks/acs.yml -i inventory_local.yml -e "@6.2.N-extra-vars.yml"`
```
Comment (alxgomz) [inline content-services/latest/install/ansible.md:262]: 6.2 is not available anymore

File: content-services/latest/install/ansible.md [hunk 6]
```diff
@@ -261,23 +309,22 @@ The diagram below shows the result of a single machine installation.
...
 ansible all -m ping -i inventory_ssh.yml
 ```
+
 To deploy different versions of Content Services use one of the following commands:
 
-* **7.2**: `ansible-playbook playbooks/acs.yml -i inventory_ssh.yml`
+* **7.2**: `ansible-playbook playbooks/acs.yml -i inventory_ssh.yml -e "@7.2.N-extra-vars.yml"`
 * **7.1**: `ansible-playbook playbooks/acs.yml -i inventory_ssh.yml -e "@7.1.N-extra-vars.yml"`
 * **7.0**: `ansible-playbook playbooks/acs.yml -i inventory_ssh.yml -e "@7.0.N-extra-vars.yml"`
 * **6.2**: `ansible-playbook playbooks/acs.yml -i inventory_ssh.yml -e "@6.2.N-extra-vars.yml"`
```
Comment (alxgomz) [inline content-services/latest/install/ansible.md:323]: same, no 6.2 anymore

File: content-services/latest/install/ansible.md [hunk 7]
```diff
@@ -295,28 +342,25 @@ transformers_1             : ok=81   changed=10   unreachable=0    failed=0    s
...
 ansible all -m ping -i inventory_ssh.yml
 ```
 
 To deploy different versions of Content Services use one of the following commands:
 
-* **7.2**: `ansible-playbook playbooks/acs.yml -i inventory_ssh.yml`
+* **7.2**: `ansible-playbook playbooks/acs.yml -i inventory_ssh.yml -e "@7.2.N-extra-vars.yml"`
 * **7.1**: `ansible-playbook playbooks/acs.yml -i inventory_ssh.yml -e "@7.1.N-extra-vars.yml"`
 * **7.0**: `ansible-playbook playbooks/acs.yml -i inventory_ssh.yml -e "@7.0.N-extra-vars.yml"`
 * **6.2**: `ansible-playbook playbooks/acs.yml -i inventory_ssh.yml -e "@6.2.N-extra-vars.yml"`
```
Comment (alxgomz) [inline content-services/latest/install/ansible.md:366]: no 6.2 anymore

**Review by alxgomz — CHANGES_REQUESTED** [review]

File: content-services/latest/install/ansible.md (hunk 7 above)
Comment (Amy-Hyland) [reply content-services/latest/install/ansible.md]: Removed 6.2

File: content-services/latest/install/ansible.md (hunk 6 above)
Comment (Amy-Hyland) [reply content-services/latest/install/ansible.md]: Removed 6.2

File: content-services/latest/install/ansible.md (hunk 5 above)
Comment (Amy-Hyland) [reply content-services/latest/install/ansible.md]: Removed 6.2

File: content-services/latest/install/ansible.md (hunk 4 above)
Comment (Amy-Hyland) [reply content-services/latest/install/ansible.md]: I've made the edits suggested.

File: content-services/latest/install/ansible.md (hunk 3 above)
Comment (Amy-Hyland) [reply content-services/latest/install/ansible.md]: I've changed the numbering as suggested.

File: content-services/latest/install/ansible.md (hunk 2 above)
Comment (Amy-Hyland) [reply content-services/latest/install/ansible.md]: I've updated the zip number.

File: content-services/latest/install/ansible.md (hunk 1 above)
Comment (Amy-Hyland) [reply content-services/latest/install/ansible.md]: I removed the CentOS 8 line.

File: content-services/latest/install/ansible.md [hunk 8]
```diff
@@ -95,72 +95,71 @@ In addition to the requirements mentioned earlier for control nodes in general,
...
 
-1. Download the Ansible playbook [zip file](https://nexus.alfresco.com/nexus/service/local/repositories/releases/content/org/alfresco/alfresco-ansible-deployment/2.0.0/alfresco-ansible-deployment-2.0.0.zip)
+1. Download the Ansible playbook [zip file](https://nexus.alfresco.com/nexus/service/local/repositories/releases/content/org/alfresco/alfresco-ansible-deployment/2.2.1/alfresco-ansible-deployment-2.2.1.zip)
 
-2. If you're not working directly on the control node, transfer the ZIP file to the control node together with the SSH private key required to login to the target machines, and SSH into the machine
+   If you're not working directly on the control node, transfer the ZIP file to  the control node together with the SSH private key required to login to the  target machines, and SSH   into the control node.
```
Comment (alxgomz) [inline content-services/latest/install/ansible.md:145]:
```suggestion
   If you're not working directly on the control node, transfer the ZIP file to  the control node together with the SSH private key required to login to the  target machines, and SSH into the control node.
```

File: content-services/latest/install/ansible.md [hunk 9]
```diff
@@ -95,72 +95,71 @@ In addition to the requirements mentioned earlier for control nodes in general,
...
-    ssh  user@controlnode
-    ```
+   ```bash
+   scp  alfresco-ansible-deployment-<version>.zip user@controlnode:
+   scp  ~/.ssh/ansible_rsa user@controlnode:.ssh
+   ssh  user@controlnode
+   ```
 
-    >**Note:** You may want to generate an SSH key pair locally and use it later for deployment. Whether you use a locally generated key, or copy over a key to the control node, it is your responsibility to deploy it to the target machines so Ansible can use it. Using SSH keys is recommended but not mandatory. If you instead use password, make sure to add the `-k`switch to the ansible command, so it prompts you for a password.
+   >**Note:** You may want to generate an SSH key pair locally and use it later  for deployment. Whether you use a locally generated key, or copy over a key to  the control node, it is your responsibility to deploy it to the target  machines so Ansible can use it. Using SSH keys is recommended but not  mandatory. If you instead use password, make sure to add the `-k` switch to  the `ansible` command, so it prompts you for a password.
```
Comment (alxgomz) [inline content-services/latest/install/ansible.md:153]:
```suggestion
   >**Note:** You may want to generate an SSH key pair locally and use it later  for deployment. Whether you use a locally generated key, or copy over a key to the control node, it is your responsibility to deploy it to the target machines so Ansible can use it. Using SSH keys is recommended but not mandatory. If you instead use password, make sure to add the `-k` switch to the `ansible` command, so it prompts you for a password.
```
I hope I'm not the one who introduced these spaces :p

File: content-services/latest/install/ansible.md [hunk 10]
```diff
@@ -395,7 +391,7 @@ There are other useful arguments you can use with the `ansible-playbook` command
 * `-K`: Prompt for sudo password. Useful when the user used to connect to the machine is not `root`.
 * `-e`: Pass an extra variable or override an existing one.
 * `-l`: Limit the play to a subset of hosts (either groups or individuals hosts or a mix of both).
-* `-u user`: specify the username to use to connect to all targets (prefer adding the `ànsible_ssh_user` to the inventory file in the right scope, e.g. under the `all`group)
+* `-u user`: specify the username to use to connect to all targets (prefer adding the `ansible_ssh_user` to the inventory file in the right scope, e.g. under the `all`group)
```
Comment (alxgomz) [inline content-services/latest/install/ansible.md:394]:
```suggestion
* `-u user`: specify the username to use to connect to all targets (prefer adding the `ansible_ssh_user` to the inventory file in the right scope, e.g. under the `all` group)
```

**Review by alxgomz — CHANGES_REQUESTED** [review]
some small things I missed in previous review

File: content-services/latest/install/ansible.md [hunk 11]
```diff
@@ -95,72 +95,71 @@ In addition to the requirements mentioned earlier for control nodes in general,
...
-| AOS | 1.5.0 | 1.4.1 | 1.4.0 |  1.4.0 | 1.3.1 |  |
-| GoogleDocs | 3.3.0 | 3.2.2 | 3.2.1 | 3.2.1 | 3.2.0 |  |
-| Digital Workspace | 3.1.0 | 2.9.0 | 2.6.0 | 2.1.0 | 2.6.0 | N/A |
-| Transform Router | 2.0.0 | 1.5.3 | 1.5.1 |1.3.2 | 1.5.1 | N/A |
-| Shared File Store | 2.0.0 | 0.16.1 | 0.16.1 | 0.13.0 | 0.16.1 | N/A |
-| Sync Service | 3.8.0 | 3.6.0 | 3.5.0 | 3.4.0 | 3.3.3.1 | N/A |
+The table below shows the version of the components deployed by the playbook for Content Services 7.x and Community.
+
+| Component | 7.3 Enterprise | 7.2 Enterprise | 7.1 Enterprise | 7.0.N Enterprise | Community |
+|-|-|-|-|-|-|
```
[the remark this answers is not in the document]
Comment (Amy-Hyland) [reply content-services/latest/install/ansible.md:122]: The latest commit should have the OS and components tables updated with what you linked. Please re-review.

**Review by alxgomz — APPROVED** [review]
LGTM

**Review by anxumalo — APPROVED** [review]
LGTM

**Pull request merged**